file/src/test/resources/sales-csv/DECIMAL.csv

```diff
@@ -0,0 +1,4 @@
+DEPTNO:int,BUDGET:decimal(18/2)
```

Comment:
The standard pattern in most sql systems is a comma between scale and precision. It's really weird that you're using a slash. Can you make this a comma?

Reply:
Done. Added double quote to disambiguate between column separator and the precision and scale separator.